root/application/src/main/java/at/fhv/itb5c/rmi/application/controller/UserController.java

```diff
@@ -0,0 +1,28 @@
+package at.fhv.itb5c.rmi.application.controller;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+
+import at.fhv.itb5c.commons.dto.IUser;
+import at.fhv.itb5c.commons.dto.IUserController;
+import at.fhv.itb5c.model.entity.User;
+import at.fhv.itb5c.rmi.application.dto.UserConverter;
+
+public class UserController implements IUserController {
+
+	@Override
+	public IUser createUser(IUser user) {
+		User userEntity = UserConverter.toEntity(user);
+
+		// TODO not nice!
+		EntityManagerFactory emf = Persistence.createEntityManagerFactory("persistence.odb");
+		EntityManager em = emf.createEntityManager();
+		em.getTransaction().begin();
+		em.persist(userEntity);
+		em.getTransaction().commit();
```

Comment:
ObjectDB documentation: An
> EntityExistsException or another PersistenceException may be thrown at flush or commit time.)
So I guess we should wrap it in a try-catch. TBD: how to react properly.